src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -666,4 +666,8 @@ Valid from:</comment>
   <data name="SignError_TimestampGeneralizedTimeInvalid" xml:space="preserve">
     <value>The timestamp's generalized time is outside the timestamping certificate's validity period.</value>
   </data>
+  <data name="InvalidPackageNupkg" xml:space="preserve">
+    <value>{0} cannot be read. File is not a valid nupkg.</value>
```

Comment:
The parameter should be at end to make the error message more readable. 

Example:

> The file is not a valid nupkg. File path: {0}